### ISSUE TITLE
running without bulk bam

### ISSUE THREAD

**Issue by isadeghi87** [lnscan/mitoSplitter#3]
Hi, 
I have a only pooled.bam, barcode and a vcf file which doesn't have mitochondrial variants. How can I run this in my situation?

best
iman

**Comment by lnscan** [comment]
您的邮件 林 已经收到，尽快回复，谢谢。

**Comment by lnscan** [comment]
Hi, iman.
I would like to know how your VCF file was generated. Are you able to calculate the mitochondrial allele frequency for each sample using other methods? If not, you may need to consider alternative demultiplexing methods. 
Because autosomes and mitochondria have different ploidy levels and the sequences are much longer, I do not recommend applying the mitoSplitter workflow directly to autosomal data.

best
Lin

**Comment by isadeghi87** [comment]
@lnscan 
So, If I have the vcf file from MT chromosome and the bam file including MT reads, can I run it?

**Comment by isadeghi87** [comment]
@lnscan

**Comment by lnscan** [comment]
Indeed, if you are able to extract MT SNPs along with their allele frequencies from a VCF file like this:
sample1 sample2 sample3
1_A 0.013779527559055118 0.018469656992084433 0.023668639053254437
1_C 0.007874015748031496 0.013192612137203167 0.007889546351084813
2_T 0.001968503937007874 0.0079155672823219 0.005917159763313609
